satellite/metainfo/metainfo.go

```diff
@@ -44,6 +44,8 @@ var (
 	mon = monkit.Package()
 	// Error general metainfo error
 	Error = errs.Class("metainfo error")
+	// ErrNodeAlreadyExists pointer already has a piece for a node err
+	ErrNodeAlreadyExists = errs.Class("metainfo error")
```

Comment:
I think this should say "err node already exists" or "metainfo error: node already exists"

Reply:
Updated to use "metainfo error: node already exists"